Backend/app/rider-platform/rider-app/Main/src/SharedLogic/GetFeedbackForm.hs

```diff
@@ -0,0 +1,98 @@
+module SharedLogic.GetFeedbackForm where
+
+import Beckn.Types.Core.Taxi.API.Rating
+import Beckn.Types.Core.Taxi.Rating
+import Beckn.Types.Core.Taxi.Rating.Category
+import Environment
+import EulerHS.Prelude hiding (id)
+import qualified Kernel.Storage.Hedis as Hedis
+import Kernel.Tools.Metrics.CoreMetrics.Types
+import Kernel.Types.Beckn.Context as Context
+import Kernel.Types.Beckn.ReqTypes
+import Kernel.Utils.Common
+import qualified SharedLogic.CallBPP as CallBPP
+import Storage.CachedQueries.CacheConfig
+
+type FeedbackFormResp = BecknReq FeedbackFormAPIEntity
+
+getFeedbackForm ::
+  ( HasFlowEnv m r ["bapSelfIds" ::: BAPs Text, "bapSelfURIs" ::: BAPs BaseUrl],
+    HasCacheConfig r,
+    Hedis.HedisFlow m r,
+    CoreMetrics m,
+    HasBapInfo r m,
+    MonadFlow m
+  ) =>
+  BaseUrl ->
+  Text ->
+  Text ->
+  Text ->
+  Int ->
+  CategoryName ->
+  m GetFeedbackFormResp
+getFeedbackForm providerUrl bppId city messageId ratingValue categoryName =
+  Hedis.safeGet (makeFeedbackFormId bppId ratingValue) >>= \case
+    Just a -> do
+      logDebug "GOT FROM REDIS"
```

Comment:
can we make message more informative or remove logs?

Reply:
removed